### PR TITLE
Add usage button to feature flags

### PR DESCRIPTION
**Pull request by timgl**
## Changes

image

One-click redirect to insights with right filters applied. Bonus:
image

We can now add `#backTo=Bla&backToURL=/some/url` anywhere to add a little 'go back to this'

## Checklist

- [ ] All querysets/queries filter by Team (if this PR affects any querysets/queries)
- [ ] Backend tests (if this PR affects the backend)
- [ ] Cypress E2E tests (if this PR affects the front and/or backend)

### CODE REVIEW

File: posthog/templates/demo.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <script>
         posthog.init('{{api_token}}', {
             api_host: window.location.origin,
-        });
+        loaded: function(posthog){posthog.isFeatureEnabled('beta-feature')}});
```
Comment (paolodamico) [inline posthog/templates/demo.html:9]: Looks like leftover code but unsure.

**Review by paolodamico — APPROVED** [review]
lgtm, love the new UX! converted the logic to kea but feel free to revert

**Comment by mariusandra** [comment]
+1 for adding stuff like this to a kea logic directly. This is something that can be useful to query inside a scene and thus could be accessible to all components. E.g. By taking the user back to the "backTo" page after a form or so.

**Comment by timgl** [comment]
Yep that's fair, thanks Paolo for doing that!

**Pull request merged**